package.json

```diff
@@ -9,8 +9,10 @@
     "prop-types": "^15.8.1",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-redux": "^8.0.1",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "redux": "^4.2.0",
     "uuid": "^8.3.2",
     "uuidv4": "^6.2.13"
   },
```

Comment:
- [ ] Kindly install the `redux toolkit` as part of your project dependencies. `npm install @reduxjs/toolkit`